### PR TITLE
docs(docker): refactor documentation to reflect submodules

### PR DESCRIPTION
**Pull request by rsdmike**
Signed-off-by: Mike <michael.johanson@intel.com>

## PR Checklist

- [x] Leverages formatting as provided by https://squidfunk.github.io/mkdocs-material/reference/abbreviations/ 
- [x] Have viewed and verified edits in rendered form ensuring proper formatting

## What are you changing?

Updating Docker setup to reflect github submodules avoiding having to worry about building each repo, and directory structure.

### CODE REVIEW

**Review by kimberlyxdavis — APPROVED** [review]
Looks good.

File: docs/Docker/dockerLocal.md [hunk 1]
```diff
@@ -1,160 +1,97 @@
...
+## Clone OpenAMT Cloud Toolkit
 
 **To clone the repositories:**
 
-1\. Open a Command Prompt or Terminal and navigate to a directory of your choice for development.
+1. Open a Command Prompt or Terminal and navigate to a directory of your choice for development.
 
 ``` bash
-git clone https://github.com/open-amt-cloud-toolkit/mps.git
+git clone --recursive https://github.com/open-amt-cloud-toolkit/open-amt-cloud-toolkit
```
Comment (bwendlandt-intel) [inline docs/Docker/dockerLocal.md:24]: Does this work with tagging? Do we need to add a '--branch v1.x.x' or some other means?

**Review by bwendlandt-intel — APPROVED** [review]
Small note on the git clone, otherwise looks great.

File: docs/Docker/dockerLocal.md (hunk 1 above)
Comment (rsdmike) [reply docs/Docker/dockerLocal.md:24]: I will update to include specific release

**Pull request merged**